pkgs/servers/jackett/default.nix

```diff
@@ -1,23 +1,32 @@
-{ lib, stdenv, fetchurl, mono, makeWrapper, curl, icu60, openssl, zlib }:
+{ lib
+, buildDotnetModule
+, fetchFromGitHub
+, dotnetCorePackages
+, openssl
+}:
 
-stdenv.mkDerivation rec {
+buildDotnetModule rec {
   pname = "jackett";
-  version = "0.18.925";
+  version = "0.19.138";
 
-  src = fetchurl {
-    url = "https://github.com/Jackett/Jackett/releases/download/v${version}/Jackett.Binaries.Mono.tar.gz";
-    sha256 = "1md0iy6sx0agsnvrj9m7bq1lvp5z34x7zv3pvwy4zw8b46w97mnz";
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
```

Comment:
Why did you do it like that? owner and repo never change, isn't it simpler to just specify "jackett"?

Reply:
Just to repeatedly specifying the same value, when it's already in a variable. `pname` shouldn't change either, so i think this should be fine.